src/plugins/displayLayout/DisplayLayoutToolbar.js

```diff
@@ -506,6 +513,47 @@ define(['lodash'], function (_) {
                     return allTelemetry;
                 }
 
+                function getToggleUnitsButton(selectedParent, selection) {
+                    let applicableItems = getAllOfType(selection, 'telemetry-view');
+                    applicableItems = unitsOnly(applicableItems);
+                    if(!applicableItems.length) {
+                        return;
+                    }
```

Comment:
I'm a big fan of short circuiting returns but since the next line is your primary return I would go with if(applicableItems.length). But, but, with the new linting rules coming up in the near future I'm not sure if it'll bark at you for only having a return be within an if.